ldk/javascript/src/webpack/generate-banner.ts

```diff
@@ -10,11 +11,47 @@ export function generateMetadata(ldkSettings: LdkSettings): string {
   if (!ldkSettings || !ldkSettings.ldk || Object.keys(ldkSettings.ldk).length === 0) {
     throw new Error(permissionsErrorMessage);
   }
+
+  if (ldkSettings.ldk.configSchema) {
+    const ajv = new Ajv();
+    const errors: string[] = [];
+
+    Object.values(ldkSettings.ldk.configSchema.properties as LdkConfigProperties).forEach(
+      (configChild) => {
+        try {
+          ajv.compile(configChild);
+
+          const { default: defaultValue, type } = configChild;
+
+          // Only allow types supported by Loop Library form
+          if (type !== 'string' && type !== 'object') {
+            errors.push(
+              `Error: The LDK does not currently support ${type} types in the config schema`,
+            );
+          }
+
+          // Check to make sure default value matches type because ajv won't
+          if (defaultValue && typeof defaultValue !== type) {
+            errors.push(`Error: The default value ${defaultValue} does not match the type ${type}`);
```

Comment:
Would be a little easier to identify the `defaultValue` if it had quotes or something!